tests/reference/cpp-print_01-026ef17.stdout

```diff
@@ -44,8 +44,8 @@ void f()
     std::cout << "a" << x << "b" << std::endl;
     x = "-+-+-";
     std::cout << "a" << x << "b" << x << "c" << std::endl;
-    std::cout << "d" << "=" << "e" << "=" << "f" << std::endl;
-    std::cout << "x" << "*\n" << "y" << "*\n" << "z" << std::endl;
+    std::cout << "d" << "=" << "e" << "=" << "f" << "+\n";
```

Comment:
Will it add an extra `+` at the end of the output? I mean the output of this generated code would be `"d=e=f+"` instead of `"d=e=f+"`.

Reply:
Yep, the tests are written that way.

Reply:
Test output:
```
Hello World!
a,b
a-+-+-b-+-+-c
d=e=f+
x*
y*
z!!1:2
LCompilers LPython
```